model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -65,6 +65,18 @@ data class AnalyzerResult(
         errors.any { it.value.isNotEmpty() }
                 || projects.any { it.scopes.any { it.dependencies.any { it.hasErrors() } } }
     }
+
+    companion object {
+        /**
+         * A constant for a [AnalyzerResult] where all properties are empty.
```

Comment:
Nit: "a*n* AnalyzerResult".

Unrelated, but what do you think about moving the companion object up to be the first statement in the class, and similar for our other companion objects that contain "val EMPTY"?

Reply:
I took `Project` as example which has it at the bottom. We should make a separate clean up commit to align this. Shall we put companion objects always to the top?

Reply:
Both done.

Reply:
Yes, I believe it makes sense to put companion objects at the top as they resemble Java static methods, and static methods usually are at the top, even before the constructor, as they can be called without constructing the object.

Reply:
Ok, I will make another PR eventually, in this class it's already fixed.